internal/worker_version_sets.go

```diff
@@ -45,10 +45,20 @@ const (
 	// VersioningIntentCompatible indicates that the command should run on a worker with compatible
 	// version if possible. It may not be possible if the target task queue does not also have
 	// knowledge of the current worker's build ID.
+	// Deprecated. This has the same effect as VersioningIntentInheritBuildID, use that instead.
```

Comment:
I think it has to be `Deprecated:` not `Deprecated.` and it may have to be in its own paragraph

https://go.dev/wiki/Deprecated

Reply:
Thanks, I'll make sure they are all properly formatted.